src/fetch.test.ts

```diff
@@ -99,8 +131,7 @@ test("fetch: should handle empty (string) responses", async () => {
 test("fetch: should handle empty (object) responses", async () => {
   const url = "https://jsonplaceholder.typicode.com/todos/empty-object";
   const resp = await fetch(url);
-
-  expect(resp.data).toEqual({});
+  expect(resp.data).toEqual("{}");
```

Comment:
We're changing the assertion from expecting an `object` to expecting a `string`. Are we sure about this?

Reply:
in this case it probably wouldn't hurt, if you provide a content type `application/json` then it will return json, if you don't it is a string. 

I added the content type so it will now return json